src/createShare.tsx

```diff
@@ -46,19 +46,18 @@ const ShareForm: React.FC<ShareFormProps> = (
 
     const [formValues, setFormState] = useState({
         endpoint: '/',
-        file_path: '/home/' + props.fileInfo.path.replace('cs3drive:', ''),
+        file_path: '/home/' + shareProps.fileInfo.path.replace('cs3drive:', ''),
```

Comment:
I'm pretty sure we should not hardcode `/home`. At CERN we still have it for now but don't use it, so I don't know if we should make this configurable. Can you check?

Reply:
But since this was already like that, I'm approving+merging and we change later.